easybuild/easyblocks/c/cuda.py

```diff
@@ -75,7 +75,7 @@ def install_step(self):
         if 'DISPLAY' in os.environ:
             os.environ.pop('DISPLAY')
 
-        run_cmd_qa(cmd, qanda, std_qa=stdqa, no_qa=noqanda, log_all=True, simple=True)
+        run_cmd_qa(cmd, qanda, std_qa=stdqa, no_qa=noqanda, log_all=True, simple=True, maxhits=1500)
```

Comment:
this allows the installer to hang without producing new output for 1500 seconds before EB gives up... isn't that *very* long?

maybe it's just a matter of fleshing out the `noqanda` a bit?

how did you see it hang exactly?